README.md

```diff
@@ -12,72 +13,90 @@ This is a set of libraries to work with [Uploadcare][1].
 
 ## Install
 
-Just update your `composer.json` with:
+Prior to installing `uploadcare-php` check if you're using
+the [Composer](getcomposer.org) dependency manager for PHP.
```

Comment:
Yup, will update the text with that, please reply the other comments first, though.